doc/WEATHER_TYPE.md

```diff
@@ -142,19 +120,38 @@ Things that weather can cause to happen.
       ],
       "spawns":
       [{
-        "max_radius":10,
-        "min_radius":10,
-        "target":"mon_zombie_survivor_elite",
-        "hallucination_count":1,
-        "real_count":0
-}]
-```  
-   ### spawn_type
-
-How many and what spawns
-
-##Fields     
-        
+        "max_radius": 10,
+        "min_radius": 10,
+        "target": "mon_zombie_survivor_elite",
+        "hallucination_count": 1,
+        "real_count": 0
+      }]
+}
+```
+
+### `requirements` properties
+
+|     Identifier                 |                              Description                              |
+| ------------------------------ | --------------------------------------------------------------------- |
+| `pressure_min`                 | |
+| `pressure_max`                 | |
+| `humidity_min`                 | |
+| `humidity_max`                 | |
+| `temperature_min`              | |
+| `temperature_max`              | |
+| `windpower_min`                | |
+| `windpower_max`                | These are all minimum and maximum values for which the weather will occur.  I.e it will only rain if its humid enough |
+| `humidity_and_pressure`        | if there are pressure and humidity requirements are they both required or just one |
+| `acidic`                       | does this require acidic precipitation                                |
+| `time`                         | Valid values are: day, night, and both.                               |
+| `required_weathers`            | a string array of possible weathers it is at this point in the loop. i.e. rain can only happen if the conditions for clouds light drizzle or drizzle are present |
```

Comment:
```suggestion
| `required_weathers`            | a string array of possible weathers that precede this one. i.e. rain can only happen if the conditions for clouds light drizzle or drizzle are present |
```
Provided "string array" is equivalent to "an array of strings". Not sure "precede" is the wording either.